### DISCUSSION TITLE
Updating to Apache 2.4.57 Win64 VS17

### DISCUSSION THREAD

**Discussion by dleffler** [leokhoa/laragon#484] [Q&A]
Quick note if you want to update to Apache 2.4.57 Win64 VS17, you'll run into issues with the Apache mod_fcgid. E.g., when you open the main Laragon page. Here's the fix

- Download the correct VS17 mod from Apache Lounge - mod_fcgid-2.3.10-win64-VS17
- Open that file and Copy the file mod_fcgid.so into the C:\laragon\etc\apache2\modules folder
- Rename mod_fcgid.so to mod_fcgid-2.3.10-win64-VS17.so

Now Laragon won't complain.

**Comment by lio60** [comment]
Buonasera, ho scaricato Laragon ho seguito una guida sul sito kreaweb.de e sono riuscito ad aggiornare phpMyAdmin ma con Apache non trovo la carteela Apache dentro la cartella bin e quando apro Laragon la voce Apache non è flaggata. Cosa devo fare? Grazie Lionello

**Reply by azri-cs** [reply]
Assuming you install Laragon on your disk C, you can simply download and use latest Apache:
1. Download the Apache binary & `mod_fcgid` module from [Apache Lounge](https://www.apachelounge.com/download/)
2. Extract the inside of the folder 'Apache24'  from the Apache 2.4.57 zip file to a new directory in _C:/laragon/bin/Apache/ for example 'httpd-2.4.57-win64-VS17'_
3. Extract the `mod_fcgid.so` file from the `mod_fcgid` zip to the directory _C:/laragon/etc/apache2/modules_ and rename the file to `mod_fcgid-2.3.10-win64-VS17.so`
4. Edit the _fcgid.conf_ file located at _C:\laragon\etc\apache2_, add this `LoadModule fcgid_module "C:/laragon/etc/apache2/modules/mod_fcgid-2.3.10-win64-VS17.so"`

**Reply by lio60** [reply]
Salve, premetto che la versione di Laragon è portatile ed è installato in M:. Ho fatto quanto chiesto ma dopo un messaggio di errore ho aggiunto "#" a "LoadModule ext_module modules/mod_fcgid.so" e quindi #LoadModule ext_module modules/mod_fcgid.so. Adesso la finestra si presenta così: 
![image](https://github.com/leokhoa/laragon/assets/81469274/3c4e8196-ae05-493a-95f9-8badb7bfdf88)
Grazie per la risposta azri-cs

**Comment by jmlocke1** [comment]
Muchas gracias por tus instrucciones, funciona perfectamente

**Comment by lio60** [comment]
Good evening, I don't know why you mentioned me in your answer, also because having no answer and in any case I don't speak English I am not at ease and I thought of shelving Laragon.

**Comment by boranbar** [comment]
**Additional Note**:

To download newest Apache versions, just use that link;
https://www.apachelounge.com/download/

**Reply by XGian71** [reply]
thanks

**Comment by alisalehi1380** [comment]
If you are using `httpd-2.4.62-240904-win64-VS17`, you need to move the files in the `Apache24 `folder to the **root directory**. You can delete the rest.

What you should end up with:
![image](https://github.com/user-attachments/assets/d976cb14-9b3c-4b0b-8759-7e2bafe09a99)

It will work properly: ✅
![image](https://github.com/user-attachments/assets/7405fb3a-710f-497f-9d50-ca76c391ac7d)
